packages/frontend/core/src/components/root-app-sidebar/index.tsx

```diff
@@ -130,6 +132,18 @@ export const RootAppSidebar = ({
     return;
   }, [onClickNewPage]);
 
+  useEffect(() => {
```

Comment:
better move this side effect to https://github.com/BABA983/AFFiNE/blob/pr/6245/packages/frontend/core/src/providers/modal-provider.tsx#L94

Reply:
✅